Classes/Emogrifier.php

```diff
@@ -181,7 +181,7 @@ public function setCss($css)
     }
 
     /**
-     * Applies the CSS you submit to the HTML you submit.
+     * Applies the CSS you submit to the HTML you submit and returns the HTML.
```

Comment:
I think the original "the CSS you submit" was not the best wording to begin with.

What about "Applies $this->css to $this->html and returns the HTML with the CSS applied."?